src/dtos/unit-fuel-base.dto.ts

```diff
@@ -1,21 +1,38 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { propertyMetadata } from '@us-epa-camd/easey-common/constants';
 import { IsInt, ValidationArguments } from 'class-validator';
-import { IsInRange } from '@us-epa-camd/easey-common/pipes';
+import { IsInRange, IsIsoFormat } from '@us-epa-camd/easey-common/pipes';
+import { IsInDbValues } from 'src/import-checks/pipes/is-in-db-values.pipe';
```

Comment:
change absolute path to relative path to prevent errors during testing. 

i.e 'src/import-checks/pipes/is-in-db-values.pipe' to '../import-checks/pipes/is-in-db-values.pipe';